easybuild/easyconfigs/g/GitPython/GitPython-3.1.0-GCCcore-8.3.0-Python-3.7.4.eb

```diff
@@ -13,6 +13,7 @@ builddependencies = [('binutils', '2.32')]
 
 dependencies = [
     ('Python', '3.7.4'),
+    ('git', '2.23.0'),
```

Comment:
@jhein32 Please use the `-nodocs` variant here too (which should fix the failing tests)

Reply:
Ok, that worked, but I have a problem with:

```sanity_pip_check = True```

If I add this line, it complains:

```AssertionError: There should be no redefined easyconfig parameters, found 1: sanity_pip_check (2)```

If I remove it, it complains:

```AssertionError: sanity_pip_check should be enabled in GitPython-3.0.3-GCCcore-8.2.0-Python-3.7.2.eb```

Please advise.  Thanks.

Reply:
It is complaining about different ECs. Open the ECs side-by-side and observe that GitPython-3.0.3-GCCcore-8.2.0-Python-3.7.2.eb does not have a `sanity_pip_check = True` line but the others do. You added it to all 3 ECs and removed it from all 3 ECs, so given that 2 out of 3 were correct already you broke either 2 or left 1 broken.

-> Add the line to GitPython-3.0.3-GCCcore-8.2.0-Python-3.7.2.eb at the same place as there is in the other 2, that is above `exts_default_options` and while you are at it add both this lines (only to that EC):

```
use_pip = True
sanity_pip_check = True
```